### PR TITLE
improve documenation

### PR DESCRIPTION
**Pull request by yelizariev**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,20 +31,27 @@ have others specific for this image.
 
 Define the cron schedule to run jobs under such circumstances.
 
-Possibly non-obvious defaults:
+Possibly non-obvious [defaults](Dockerfile):
```
Comment (Yajo) [inline README.md]: I'm not sure this link will work, as the README can be seen from several places. Better add a full URL please.

```suggestion
Possibly non-obvious [defaults](https://github.com/Tecnativa/docker-duplicity/blob/master/Dockerfile):
```

File: README.md [hunk 2]
```diff
@@ -31,20 +31,27 @@ have others specific for this image.
...
+If you set these values in `.env` file, don't use quotes:
+
+```
+CRONTAB_15MIN=*/15 * * * *
+CRONTAB_HOURLY=0 * * * *
+CRONTAB_DAILY=0 2 * * MON-SAT
+CRONTAB_WEEKLY=0 1 * * SUN
+CRONTAB_MONTHLY=0 5 1 * *
+```
 ### `DST`
```
Comment (Yajo) [inline README.md]:
```suggestion

### `DST`
```

File: README.md [hunk 3]
```diff
@@ -89,7 +96,24 @@ Define when to execute the command you defined in the previous section. If you
 need several values, you can separate them with spaces (example: `daily
 monthly`).
 
-Check the `Dockerfile` to see built-in jobs.
+[Built-in jobs](Dockerfile):
```
Comment (Yajo) [inline README.md]: Same as above.
```suggestion
[Built-in jobs](https://github.com/Tecnativa/docker-duplicity/blob/master/Dockerfile):
```

File: README.md [hunk 4]
```diff
@@ -89,7 +96,24 @@ Define when to execute the command you defined in the previous section. If you
...
+
+```
+# Makes postgres dump for all databases. Uploads dumps and all other files
```
Comment (Yajo) [inline README.md]: This only applies to the postgres flavors. Please move these docs to that section.

File: README.md [hunk 5]
```diff
@@ -89,7 +96,24 @@ Define when to execute the command you defined in the previous section. If you
...
+# Available in all images
+JOB_200_WHEN=daily weekly
+
+# Incremental backup of all files
+# Available in postgres, postgres-s3 images
```
Comment (Yajo) [inline README.md]: Same as above.

File: README.md [hunk 6]
```diff
@@ -89,7 +96,24 @@ Define when to execute the command you defined in the previous section. If you
...
+JOB_300_WHEN=daily
+
+# Full backup of all files
+# Available in all images besides "tecnativa/duplicity:docker"
+JOB_500_WHEN=weekly
```
Comment (Yajo) [inline README.md]: Actually these are the only jobs available to all flavors:

https://github.com/Tecnativa/docker-duplicity/blob/8e4e182df50598cea16cc0a2ea7c6109b2b6dd14/Dockerfile#L14-L15

Weekly full backups are available to all s3 flavors. Move this info there.

File: README.md [hunk 7]
```diff
@@ -89,7 +96,24 @@ Define when to execute the command you defined in the previous section. If you
...
+```
+
+
+You can disable those jobs by setting corresponding `JOB_*_WHEN` to value `never`.
```
Comment (Yajo) [inline README.md]: Cool hack!

File: README.md [hunk 8]
```diff
@@ -181,6 +205,8 @@ You can use these bundled binaries to work faster:
 - `backup`: Executes an immediate backup with default options.
 - `restore`: Restores immediately with default options. Most likely, you will
   need to use it with `--force`.
+- `/etc/periodic/daily/jobrunner`: execute immediately all jobs scheduled for daily backups.
+
```
Comment (Yajo) [inline README.md]:
```suggestion
  Change `daily` for other periodicity if you want to run those instead.
```

**Comment by yelizariev** [comment]
Sorry for the delay. It's ready to review

File: README.md [hunk 9]
```diff
@@ -297,11 +317,17 @@ services:
 
 Any of the other flavors has a special variant suffixed with `-s3`. It
 provides some opinionated defaults to make good use of S3 different storage
-types and its lifecycle rules and filters, assuming you want to have weekly
-full backups. You should combine it with lifecycle and expiration rules at
-your will.
+types and its lifecycle rules and filters, assuming you want to have 
+[weekly full backups][Dockerfile]. You should combine it with lifecycle and 
+expiration rules at your will.
```
Comment (Yajo) [inline README.md]: Would this be a good spot to document the kind of S3 URL you should use? As discussed in https://github.com/Tecnativa/doodba-scaffolding/pull/64

Comment (yelizariev) [reply README.md]: Done. Check the latest version

**Pull request merged**